libmamba/include/mamba/core/output.hpp

```diff
@@ -12,6 +12,7 @@
 #include <string>
 #include <string_view>
 #include <vector>
+#include <iostream>
```

Comment:
I think this should be `iosfwd` as it is in headers and we want to avoid bring unnecessary heavy headers.